distro/tomcat/assembly/src/conf/bpm-platform.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <bpm-platform xmlns="http://www.operaton.org/schema/1.0/BpmPlatform" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xsi:schemaLocation="http://www.operaton.org/schema/1.0/BpmPlatform http://www.operaton.org/schema/1.0/BpmPlatform ">
+  xsi:schemaLocation="http://www.operaton.org/schema/1.0/BpmPlatform https://www.operaton.org/schema/1.0/BpmPlatform ">
```

Comment:
As noted in the issue, do not change any namespace URI. Please revert all changes of that kind.

Reply:
reverted